app1-2-js.js

```diff
@@ -0,0 +1,9 @@
+const quiz =['question','answears'];
+const answers=true;
```

Comment:
下記のように`=`の両外側にはスペースを空けましょう。
```suggestion
const answers = true;
```